libs/components/core/src/lib/modules/percent-pipe/fixtures/percent-pipe.component.fixture.ts

```diff
@@ -7,9 +7,11 @@ import { SkyPercentPipe } from '../percent.pipe';
   templateUrl: './percent-pipe.component.fixture.html',
 })
 export class PercentPipeTestComponent {
-  public numberValue: any = 0.8675309;
-  public format: string;
-  public locale: string;
+  public format: string | undefined;
+
+  public locale: string | undefined;
+
+  public numberValue: unknown = 0.8675309;
```

Comment:
Why is this declared as `unknown` when it has an initial value?

Reply:
Added a comment above explaining why.